source/renderer/app/themes/utils/findUpdates.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-console */
+// @flow
+import { isEmpty } from 'lodash';
+import chalk from 'chalk';
+import { findMissingDefinitions, findMissingCSSVars } from './checkCreateTheme';
+import { CARDANO_THEME_CONFIG } from '../daedalus/cardano';
+import { DARK_BLUE_THEME_CONFIG } from '../daedalus/dark-blue';
+import { LIGHT_BLUE_THEME_CONFIG } from '../daedalus/light-blue';
+import type {
+  PendingThemesUpdates,
+  FindUpdatesParams,
+  LogDifferencesParams,
+} from '../types';
+
+const logDifferences = ({
+  color,
+  missingDefs,
+  themeName,
+}: LogDifferencesParams) => {
+  // $FlowFixMe
+  const message = chalk`\n{underline ${themeName}} is missing the following definitions that exist on the {bold createTheme} object:\n\n${JSON.stringify(
+    missingDefs,
+    0,
+    2
+  )}\n`;
+  return console.log(chalk.hex(color)(message));
+};
+
+// Checks for properties/CSS vars on createThemeObj that don't exist on existing themes
+export const findUpdates = (
+  createThemeOutputs: FindUpdatesParams
+): null | PendingThemesUpdates => {
+  const { cardano, darkBlue, lightBlue } = createThemeOutputs;
+  const pendingThemesUpdates = {};
+
```

Comment:
@MarcusHurney  Also suggestion to automatise defsToAdd regarding to passed params "${prefix}_THEME_CONFIG" as collection instead manual creation for each of them.

Reply:
I agree! 👍